syz-cluster/controller/api.go

```diff
@@ -55,6 +57,23 @@ func (c ControllerAPI) getSessionSeries(w http.ResponseWriter, r *http.Request)
 	reply(w, resp)
 }
 
+func (c ControllerAPI) skipSession(w http.ResponseWriter, r *http.Request) {
+	req := parseBody[api.SkipRequest](w, r)
+	if req == nil {
+		return
+	}
+	ctx := context.Background()
+	err := c.seriesService.SkipSession(ctx, r.PathValue("session_id"), req)
```

Comment:
You can use r.Context() here.

Reply:
Good point! Patched it everywhere.

Reply:
It is still `ctx := context.Background()`.